src/core/modules/AtmosphereShowMySky.cpp

```diff
@@ -724,12 +730,40 @@ void AtmosphereShowMySky::computeColor(StelCore* core, const double JD, const Pl
 	{
 		lastUsedAltitude_ = location.altitude;
 		probeZenithLuminances(location.altitude);
+		dynResTimer=0;
 	}
 
 	auto sunPos  =  sun.getAltAzPosAuto(core);
 	if (std::isnan(sunPos.length()))
 		sunPos.set(0.,0.,-1.*AU);
 
+	if (flagDynamicResolution)
+	{
+		float f1=prj->getFov(), i1=fader.getInterstate();
```

Comment:
I'd use more descriptive names than `<letter><digit>`.

Reply:
BTW

> `sunPos.set(0.,0.,-1.*AU);`

> `// If we have no moon, just put it into nadir at 1 AU distance`
> `Vec3d moonPos{0.,0.,-1.*AU};`
`

Aren't we already scaled to multiples of AU?

Reply:
Hmm, indeed there's an inconsistency in units here. It just doesn't make much difference since these are cases where the Sun or the Moon aren't visible.

Reply:
Funny, this multiplication by `AU` dates back to 2009: 9d6b8ada86505bebb95350b134596a1d8db4cabc.